### PR TITLE
Some pre 2 mappings

### PR DESCRIPTION
**Pull request by modmuss50**
Not too sure on DimensionData as a class name, please let me know if you have any better ideas.

### CODE REVIEW

File: mappings/net/minecraft/util/registry/SimpleRegistry.mapping [hunk 1]
```diff
@@ -5,4 +5,8 @@ CLASS net/minecraft/class_2370 net/minecraft/util/registry/SimpleRegistry
 	FIELD field_11110 indexedEntries Lnet/minecraft/class_3513;
 	FIELD field_11111 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_25067 entriesByKey Lcom/google/common/collect/BiMap;
+	FIELD field_25489 registryKeys Ljava/util/Set;
 	METHOD method_10240 getRandom (Ljava/util/Random;)Ljava/lang/Object;
+	METHOD method_29722 getEntriesByKey ()Ljava/util/Set;
+	METHOD method_29723 containsRegistryKey (Lnet/minecraft/class_5321;)Z
+	METHOD method_29725 addRegistryKey (Lnet/minecraft/class_5321;)V
```
Comment (liach) [inline mappings/net/minecraft/util/registry/SimpleRegistry.mapping]: This is actually marking certain keys as loaded from encoder than builtin from code. Addressed in my pr

File: mappings/net/minecraft/util/registry/Registry.mapping [hunk 2]
```diff
@@ -102,6 +102,7 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_25296 LOOT_FUNCTION_TYPE_KEY Lnet/minecraft/class_5321;
 	FIELD field_25297 LOOT_CONDITION_TYPE_KEY Lnet/minecraft/class_5321;
 	FIELD field_25299 LOOT_CONDITION_TYPE Lnet/minecraft/class_2378;
+	FIELD field_25490 DIMENSION_DATA Lnet/minecraft/class_5321;
```
Comment (liach) [inline mappings/net/minecraft/util/registry/Registry.mapping]: Wonder about this name... a combination of chunk generator and dimension type

Comment (modmuss50) [reply mappings/net/minecraft/util/registry/Registry.mapping]: What about DimensionOptions?

Comment (liach) [reply mappings/net/minecraft/util/registry/Registry.mapping]: that actually makes sense as that is the sub option in generator options :+1:

**Comment by liach** [comment]
Mind edit pr to merge into pre2

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**